bower.json

```diff
@@ -75,9 +75,6 @@
     "knockout": {
       "js": "dist/knockout.debug.js"
     },
-    "libsodium.js": {
```

Comment:
right.. :) I recently started from scratch and was getting this error during bower install.. works now with another clean start.